connectors/connectors-common-library/src/main/java/io/camunda/connector/common/model/CommonResult.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. Licensed under a proprietary license.
+ * See the License.txt file for more information. You may not use this file
+ * except in compliance with the proprietary license.
+ */
+package io.camunda.connector.common.model;
+
+import com.google.common.base.Objects;
+import java.util.Map;
+
+public class CommonResult {
```

Comment:
I'd probably name it `CommonResponse` to adhere to the common HTTP terminology. But I'm fine with `CommonResult` as well if you think it's better.

Reply:
That would be logical, on the other hand though we have SomethingRequest and SomethingResult in every single connector except slack. The original name for the http result object is also HttpJsonResult. But it is a really small change so I would say it is up to you. If you still think CommonResult would be better I can change it easily.

Reply:
Valid point. I'm totally fine with `CommonResult` then.